reference/7.1/Microsoft.PowerShell.Core/About/about_Split.md

```diff
@@ -131,10 +131,13 @@ the substrings are returned. A value of 0 returns all the substrings. Negative
 values return the amount of substrings requested starting from the end of the
 input string.
 
-Max-substrings does not specify the maximum number of objects that are
-returned; its value equals the maximum number of times that a string is split.
-If you submit more than one string (an array of strings) to the Split operator
-, the Max-substrings limit is applied to each string separately.
+> [!NOTE]
+> Support for negative values was added in PowerShell 7.
+
+**Max-substrings** does not specify the maximum number of objects that are
+returned. its value equals the maximum number of times that a string is split.
```

Comment:
`Its` should be capitalized since the semi-colon was replaced with a comma.

Reply:
Fixed